pennylane/ops/qubit/parametric_ops.py

```diff
@@ -40,7 +40,7 @@ def _can_replace(x, y):
     Convenience function that returns true if x is close to y and if
     x does not require grad
     """
-    return (not qml.math.requires_grad(x)) and qml.math.allclose(x, y)
+    return not qml.math.is_abstract(x) and not qml.math.requires_grad(x) and qml.math.allclose(x, y)
```

Comment:
Why is this needed? Is there a rule of thumb in which cases user code might require changes like this?

Reply:
Because you are not supposed to be able to compare the value of jitted operations, previsouly a jitted parameter would return `requires_grad=True` if without being trainable, so I had to add the check.